test/Nancy.Tests/project.json

```diff
@@ -2,8 +2,8 @@
     "version": "1.4.1.0",
     "dependencies": {
         "dotnet-test-xunit": "2.2.0-preview2-build1029",
-        "FakeItEasy": "3.0.0-rc001",
-        "FakeItEasy.Analyzer": "3.0.0-rc001",
+        "FakeItEasy": "3.0.0",
+        "FakeItEasy.Analyzer": "3.0.0",

```

Comment:
It tells you when you've an unused call specification, like `A.CallTo(()=>fake.Method());` with no instructions for the fake, and it also warns if you try to configure a method that is not virtual (the latter would be caught at runtime as well, but why not point out misuse as early as possible?).

See http://fakeiteasy.readthedocs.io/en/stable/analyzer/ for more.

Hmm. On closer inspection, it says nearly exactly that, so maybe save yourself a click.